spec/tests/PlayerEvents.spec.ts

```diff
@@ -196,13 +196,15 @@ describe('player event tests', () => {
           playerEventHelper.fireSeekEvent(50.145);
           expect(MockHelper.latestVideoAnalytics.reportPlaybackMetric).toHaveBeenCalledWith(
             Conviva.Constants.Playback.SEEK_STARTED,
+            expect.any(Number)
           );
         });
 
         it('on timeshift', () => {
           playerEventHelper.fireTimeShiftEvent();
           expect(MockHelper.latestVideoAnalytics.reportPlaybackMetric).toHaveBeenCalledWith(
             Conviva.Constants.Playback.SEEK_STARTED,
+            -1
```

Comment:
Why `-1`? 🤔

Reply:
It was this way always:

<img width="588" alt="image" src="https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/assets/85180621/b50cc9c8-eb1f-4054-b668-1088659e0fd7">

But there was a problem with passing this value around in the previous implementation, so it's fixed here.

Reply:
Thanks for clarifying!